Google.PowerShell.IntegrationTests/BigQuery/BigQuery.BqTable.Tests.ps1

```diff
@@ -136,22 +136,85 @@ Describe "New-GcbqTable" {
 
     It "should properly set the duration of time for which the tables last" {
         $expireInSec = 3600
-        $table = New-GcbqTable -DatasetId $datasetName "my_table_duration" -Expiration $expireInSec
+        $table = New-BqTable -DatasetId $datasetName "my_table_duration" -Expiration $expireInSec
         $calculatedTime = [int64](([datetime]::UtcNow)-(get-date "1/1/1970")).TotalMilliseconds
         $table.ExpirationTime | Should BeLessThan ($calculatedTime + (($expireInSec + 5) * 1000))
     }
 
     It "should throw when there is already a table with the same ID"{
-        New-GcbqTable "my_table_repeat" -DatasetId $datasetName 
-        { New-GcbqTable "my_table_repeat" -DatasetId $datasetName -ErrorAction Stop } | Should Throw "409"
+        New-BqTable "my_table_repeat" -DatasetId $datasetName 
+        { New-BqTable "my_table_repeat" -DatasetId $datasetName -ErrorAction Stop } | Should Throw "409"
     }
 
     It "should throw when the dataset is not found"{
-        { New-GcbqTable "my_table_d404" -DatasetId $nonExistDataset } | Should Throw "404"
+        { New-BqTable "my_table_d404" -DatasetId $nonExistDataset } | Should Throw "404"
     }
 
     It "should throw when the project is not found"{
-        { New-GcbqTable -DatasetId $datasetName "my_table_p404" -project $nonExistProject} | Should Throw "404"
+        { New-BqTable -DatasetId $datasetName "my_table_p404" -project $nonExistProject} | Should Throw "404"
+    }
+
+    AfterAll {
+        $test_set | Remove-BqDataset -Force
+    }
+}
+
+Describe "Remove-BqTable" {
+
+    BeforeAll {
+        $r = Get-Random
+        $datasetName = "pshell_testing_$r"
+        $test_set = New-BqDataset $datasetName
+    }
+
+    It "should not delete the table if -WhatIf is specified" {
+        $table = New-BqTable -Dataset $test_set -Table "table_if"
```

Comment:
I randomize the datasetId and each table is named for the test case that it's in.  When each set of tests is finished, the dataset is deleted, so there should never be collisions.  I'm not opposed to adding it, but I think it'll keep the test cases cleaner and more concise without it.